docs/guides/qiskit-addons-mpf-get-started.ipynb

```diff
@@ -0,0 +1,372 @@
+{
```

Comment:
<p>I am not a particular fan of this phrasing:</p><p> </p><blockquote>It does so by executing several circuits, each of which evolves the system with a different number of Trotter steps, and recombines the results into a single measurement.</blockquote><p> </p><p>The package itself does neither the execution nor do the results get combined into a single "measurement".</p><p>Can we instead emphasize that it provides the tools for determining the weights that allow one to recombine the expectation values from multiple Trotter circuits?</p><p> </p><p>------</p><p> </p><p>There is also a problem in the first equation. The 10 needs to be wrapped in brackets in order to ensure the superscript works correctly.</p><p>The description of $J$ and $h$ in the following sentence are inconsistent with regards to including the index subscript. I am fine with either keeping it for both or neither.</p><p> </p><p>The total magnetization should also span all 10 sites and not only 9.</p><p> </p><p>The utils package is called <code>qiskit-addon-utils</code> (not <code>addons</code> ). </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/2367/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>